### ISSUE TITLE
Pay attention to more sensors

### ISSUE THREAD

**Issue by Krutonium** [Krutonium/BetterFanController#17]
At this point we pay attention to only one sensor on the GPU, but for example on my 5600XT, there is 3 - The one we pay attention to, Edge, Junction, and Memory.

I believe that the automatic fan control also makes this mistake, both on Linux and Windows, because those two components both get significantly hotter than the edge temp, and they can/will overheat long before the edge temp indicates an issue, leading the the hardware failures that are common the RX 5000 series of GPU's - Setting the fans to 100% at 50c on edge alleviates the issue but not intentionally.

Basically we should probably be moderating the fans based on all those values, not just the edge temp.
